uaclient/util.py

```diff
@@ -299,8 +299,10 @@ def get_machine_id(cfg) -> str:
     # Docker images do not define ETC_MACHINE_ID or DBUS_MACHINE_ID on trusty
     # per Issue: #489
 
-    if cfg.machine_token:
-        cfg_machine_id = cfg.machine_token.get("machineId")
+    if isinstance(cfg.machine_token, dict):
```

Comment:
Why we need the `isinstance` check here ?

Reply:
we don't: there was a failing test, but its related to some mock behavior. Already removed.